content/working-sessions/getting-more-women-in-cyber-security.md

```diff
@@ -4,21 +4,21 @@ type         : working-session
 status       : draft
 organizers   : 
 description  :
----
```

Comment:
Please don't delete these hyphens: they are needed. Otherwise, the page will not show!